core/src/main/scala/org/broadinstitute/dsde/rawls/entities/compact/CompactEntityProviderConfig.scala

```diff
@@ -1,7 +1,6 @@
 package org.broadinstitute.dsde.rawls.entities.compact
 
 case class CompactEntityProviderConfig(
-  // Controls approximately how large a single batch-insert SQL statement will be for the batchUpsert/batchUpdate APIs.
-  // Default is 100 Mb, well under the 1Gb packet size we have set for MySQL
-  maxSqlBatchSizeBytes: Int = 100 * 1024 * 1024
+  // number of entities to handle in a single SQL statement when processing batchUpsert/batchUpdate
+  batchUpsertBatchSize: Int = 2500
```

Comment:
This is isolated/encapsulated into this config class to make it easier to move to rawls.conf if we ever feel the need to do so